router.go

```diff
@@ -953,6 +978,56 @@ func Router(addr string) *echo.Echo {
 		g.GET("/:app/:version/tarball/:tarball", getVersionTarball)
 	}
 
+	virtuals := config.GetConfig().VirtualSpaces
+	for name, virtual := range virtuals {
+		groupName := fmt.Sprintf("/%s/registry", url.PathEscape(name))
+		g := e.Group(groupName, ensureSpace(virtual.Source))
+
+		// TODO
+		g.GET("", getAppsList, jsonEndpoint, middleware.Gzip())
```

Comment:
This route is harder to filter, but I think it's possible to code it in a single day